components/html-block/README.md

```diff
@@ -56,23 +56,23 @@ Examples are provided to display how user-authored math can be embedded within y
   import '@brightspace-ui/core/tools/mathjax-test-context.js';
 </script>
 <d2l-html-block html="
-  <math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;>
-    <msqrt>
-      <mn>3</mn>
-      <mi>x</mi>
-      <mo>&#x2212;</mo>
-      <mn>1</mn>
-    </msqrt>
-    <mo>+</mo>
-    <mo stretchy=&quot;false&quot;>(</mo>
-    <mn>1</mn>
-    <mo>+</mo>
-    <mi>x</mi>
-    <msup>
-      <mo stretchy=&quot;false&quot;>)</mo>
-      <mn>2</mn>
-    </msup>
-  </math>">
+  &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;>
```

Comment:
This makes the README look pretty unpleasant, unfortunately, but I believe it's necessary to fix the way the html-block is mangled in the Daylight site.

Reply:
I wonder if we can set it up to use `render` and an encoding statement, so that the MathML can be presented in an unencoded state? Or, should we even present the `slot` option?

Reply:
Yeah it's kinda ugly. Should we be presenting this in an unencoded state, and using an encoding method?

Reply:
Where would you propose doing the encoding? I would be a bit nervous about integrating something like that into the html-block, given that there's no built-in browser API for this.

Reply:
If they pass it as an attribute, it looks like it would have to be HTML encoded, so maybe somewhere in the demo. If they pass it to the JavaScript property, does it have to be encoded also?

I guess what I'm getting at is, we should demo what they are expected to do.

Reply:
Right, somewhere in the demo would make more sense to me. I'm not 100% sure if the same would be necessary if passed in as a property; this is one of those cases where the browser seems to mostly figure things out regardless, but the demo component in particular is struggling to parse things. i.e. I'm not really sure if it's the browser, our own code, or Prism's code choking (I didn't really look into exactly where the mangling was sourced from).

Reply:
It's worth noting though that in the html-block's demo page, this all renders fine (the only visibly escaped characters are the quotes). Of course, that's not true in the README because of (presumably) however GitHub is massaging the markdown into HTML. I'm not 100% sure if making that change in the demo pages is going to mess up the fact that we kind of have to escape the quotes for the rendering to work in the first place.

Reply:
(I'm going to merge this PR for now either way, since I think if we decide to action something for this encoding problem, that can happen independently of this.)